### PR TITLE
Add Python test tool Nox

### PR DESCRIPTION
**Pull request by hugovk**
Support added in https://github.com/wntrblm/nox/releases/tag/2022.1.7

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ Many software programs already use `--color` or `-c` to enable color output. Thi
 | Software | Description | Date / Version Supported |
 |-|-|-|
 | [node](https://nodejs.org/node) | JavaScript runtime | |
+| [Nox](https://github.com/wntrblm/nox) | Flexible test automation | [2022-01-07 / 2022.1.7](https://github.com/wntrblm/nox/releases/tag/2022.1.7) |
 | [npm](https://www.npmjs.com/npm) | Package manager for JavaScript | |
 | [jest](https://jestjs.io/jest) | JavaScript testing framework | |
 | [nox](https://nox.thea.codes) | Python automation | [2022-01-07 / 2022.1.7](https://github.com/wntrblm/nox/releases/tag/2022.1.7) |
```
Comment (donatj) [inline index.md]: I've already got nox in here, coule lines down.

Comment (hugovk) [reply index.md]: Oops, sorry for the duplicate! 

Let's capitalise the name, like the project does, and put them in alphabetical order.

I'll update the PR.

File: index.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ Many software programs already use `--color` or `-c` to enable color output. Thi
 | Software | Description | Date / Version Supported |
 |-|-|-|
 | [node](https://nodejs.org/node) | JavaScript runtime | |
+| [Nox](https://github.com/wntrblm/nox) | Flexible test automation | [2022-01-07 / 2022.1.7](https://github.com/wntrblm/nox/releases/tag/2022.1.7) |
```
Comment (donatj) [inline index.md]: Thoughts or objections on simply adding `for Python` - I think it's helpful to include for people what language things are for incase they are looking for something they can use.

```suggestion
| [Nox](https://github.com/wntrblm/nox) | Flexible test automation for Python | [2022-01-07 / 2022.1.7](https://github.com/wntrblm/nox/releases/tag/2022.1.7) |
```

Comment (hugovk) [reply index.md]: Good idea!

**Comment by donatj** [comment]
Thanks once again!

**Pull request merged**